### PR TITLE
New Port ElecHead

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for ElecHead

## Game Information
- **Title**: ElecHead
- **URL**: https://namatakahashi.itch.io/elechead

### CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] ROCKNIX
[X] MuOS

Architectures: 
[X] aarch64
[X] Panfrost (Mainline)

Resolutions: 
[X] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/elechead/port.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "ElecHead.sh",
+    "elechead"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "ElecHead",
+    "porter": [
+      "Fraxinus88"
+    ],
+    "desc": "⚡＋🤖＋🧩",
```
Comment (kloptops) [inline ports/elechead/port.json:14]: Not sure this will work. 😁

File: ports/elechead/cover.png
Comment (kloptops) [inline ports/elechead/cover.png:1]: This is the same as the screenshot?

**Review by kloptops — CHANGES_REQUESTED** [review]

File: ports/elechead/cover.png
Comment (Fraxinus88) [reply ports/elechead/cover.png:1]: Whoops

File: ports/elechead/port.json (hunk 1 above)
Comment (Fraxinus88) [reply ports/elechead/port.json:14]: Can we try :p

Comment (kloptops) [reply ports/elechead/port.json:14]: Sure. :)

**Review by kloptops — APPROVED** [review]
Good luck. Let me know when you want to update the port.json file with real info. 😄

**Review by Cebion — APPROVED** [review]

**Pull request merged**